media/inf/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ list(APPEND INF_FILES
     fdc.inf
     flpydisk.inf
     hal.inf
-    hdaudbus.inf
+    #hdaudbus.inf
```

Comment:
This and the other change was made to a local branch, I don't know how these changes were merged to the translations branch

Reply:
A way to fix this problem is by rebasing the `Spanish-translations` branch and dropping `
006e203` which is the commit unrelated to your PR. However, keep in mind you've got to forcefully push the branch with the `--force-with-lease` command line option in order for changes to take effect.

Here is a step-by-step explanation if you did not understand my comment above.

**1.**  First of all, we must make sure that we are in the specific branch. If not, type `git checkout Spanish-translations` within the command line prompt so that you'll switch from the `master` (or the previous branch) into the specified local branch.

**2.** To rebase the branch, type the following command line `git rebase -i HEAD~2`. The `-i` flag means interactive which it'll open a text editor with a list of commits and a plethora of options like dropping, squashing, editing, etc. The `2` number is the number of commits of your branch (assuming your branch already has two pushed/non-pushed commits).

**3.** Our troublesome commit is `006e203` so we've got to remove it from the branch. At the front of the hash ID of the commit, replace `pick` with `drop` (or just `d`, your choice). Save the changes and exit from the text editor. 

**4.** We're done with rebasing, however the changes are not completely pushed globally. To update your branch, using `git push --force-with-lease` is the good way to go.

Reply:
I definitively give up with github
![imagen](https://user-images.githubusercontent.com/14007379/42486934-33096446-83c4-11e8-8207-7462d2672401.png)

Reply:
You're missing the "force push" option. I'm not sure where that's located in Git Gui. I can fix it for you.